release.md

```diff
@@ -1,30 +1,48 @@
-- Include the relevant changelog entries.
-- Don't forget to check "This is a pre-release".
+Perform the release procedure on GitHub: [Create a new release](https://github.com/lumicks/pylake/releases/new):
+- Create a new tag `v<release_version_number>` with target `main`. 
+- Set the release title to `v<release_version_number>`.
+- Include the relevant changelog entries into the description.
+- Check "This is a pre-release".
+- Wait until all [actions](https://github.com/lumicks/pylake/actions) of the release merge have passed.
+- "Publish release".
 
 ### Conda Forge
 
-- Go to the PyPI page of the newly released version: https://pypi.org/project/lumicks.pylake/
-- Under download files, check the sha256 hash of the tarball. You will need this for releasing on Conda Forge.
-- Use our [conda-forge feedstock fork](https://github.com/lumicks/lumicks.pylake-feedstock) and create a new branch based on `upstream/master`.
-- Create a new branch on our feedstock for your release.
-- Update the version in the `recipe/meta.yaml` if the version number changed. If the build number is not zero, set it to zero. Also update the sha256 hash.
-- If it's a re-release because some dependencies changed, increment the build number but do not change the version.
-See https://conda-forge.org/docs/maintainer/updating_pkgs.html for more info.
-- Make a PR on the [upstream conda-forge repository](https://github.com/conda-forge/lumicks.pylake-feedstock). Wait for the tests to complete and pull in the changes.
+- Wait until the new release of [Pylake has been published on PyPI](https://pypi.org/project/lumicks.pylake/)
+- Copy the sha256 hash of the tarball under [download files](https://pypi.org/project/lumicks.pylake/#files). You will need this for releasing on Conda Forge.
+
+- Update the Pylake conda package hosted on the `conda-forge` channel. For details see [Maintaining packages](https://conda-forge.org/docs/maintainer/updating_pkgs.html):
+  - Clone [our conda-forge feedstock fork](https://github.com/lumicks/lumicks.pylake-feedstock)
+  - On main `git rebase upstream/main` (see [2. Syncing your fork with conda-forges feedstock](https://conda-forge.org/docs/maintainer/updating_pkgs.html#example-workflow-for-updating-a-package)
+  - Create a new branch `git checkout -b release_v<X_YY_Z>`
+  - Update `recipe/meta.yaml`:
+    - Update "version" if the Pylake version number changed.
+    - Update the sha256 hash with the copy from PyPI.
+    - Ensure the build number is zero, or if it is a re-release because some dependencies changed, increment the build number (but do not change the Pylake version).
+    - Check if your github username is listed under `recipe-maintainers`
+  - Push the new branch to our conda-forge feedstock fork: `git push -u origin release_v<X_YY_Z>`
+  - If you had to add yourself as a `recipe-maintainer`, let _another_ maintainer do the PR in the next step, as you do not yet have sufficient rights to trigger a `rerender`!
```

Comment:
I don't think that was the reason it failed. I think the reason was that the conda bot didn't have permissiont to push to the PR coming from our fork. Normally you can give it permission by checking "Allow maintainers to push" but this checkbox doesn't exist when you open a PR from our feedstock since the repo is on an org account rather than your personal one. I suspect currently, the only method that works is to locally rerender with `conda-smithy`. For now, I would refer to the conda docs section for this. Next time we can further refine this point.

See also: https://github.com/conda-forge/webservices-dispatch-action/issues/4

Reply:
Ah, OK. Changed it and included a link to the docs.